README.md

```diff
@@ -23,18 +23,18 @@ npm i --save @edx/gradebook
 
 After cloning the repository, run `make up-detached` in the `gradebook` directory - this will build and start the `gradebook` web application in a docker container.
```

Comment:
The `up-detached` command was deleted, so this line should be updated.

Reply:
good call